RevenueCatUI/CustomerCenter/ViewModels/ActiveSubscriptionsListViewModel.swift

```diff
@@ -48,12 +56,16 @@ final class ActiveSubscriptionsListViewModel: BaseManageSubscriptionViewModel {
     // Used for Previews
     convenience init(
         screen: CustomerCenterConfigData.Screen,
-        activePurchases: [PurchaseInformation] = []
+        activePurchases: [PurchaseInformation] = [],
+        originalAppUserId: String = UUID().uuidString,
+        originalPurchaseDate: Date? = Date()
```

Comment:
Hmm I wonder if these should have these defaults... I think it would be preferable to make them nullable and default to nil? Or make them mandatory parameters so we don't forget passing them.